src/main/java/ru/kontur/airlock/Validator.java

```diff
@@ -14,6 +14,10 @@ public boolean validate(String eventRoutingKey) {
         return validateForbiddenCharacters(eventRoutingKey) && validateApiKeyAccess(eventRoutingKey);
     }
 
+    public boolean validateApiKey() {
```

Comment:
validateApiKey() is unclear method, I should say that nullable array is unacceptable at Validator.

Reply:
I did not write this class and this out of pull request scope